README.md

```diff
@@ -9,15 +9,15 @@ This is a template repository to kickstart the creation of a Python Vehicle App.
 This repository provides you with a complete development environment for your own Vehicle App based on the [Software defined vehicle platform](https://sdv.eclipse.org/) including a sample Vehicle App using the [Vehicle App Python SDK](https://github.com/eclipse-velocitas/vehicle-app-python-sdk). The development environment uses the [Development Container](https://code.visualstudio.com/docs/remote/create-dev-container#:~:text=%20Create%20a%20development%20container%20%201%20Path,additional%20software%20in%20your%20dev%20container.%20More%20) feature of Visual Studio Code.
 
 ## Documentation
-* [Velocitas Development Model](https://symmetrical-waddle-dc3368d3.pages.github.io/docs/concepts/development-model/)
-* [Vehicle App Python SDK Overview](https://symmetrical-waddle-dc3368d3.pages.github.io/docs/python-sdk/python_vehicle_app_sdk_overview/)
+* [Velocitas Development Model](https://eclipse-velocitas.github.io/velocitas-docs/docs/concepts/development_model/)
+* [Vehicle App Python SDK Overview](https://eclipse-velocitas.github.io/velocitas-docs/docs/reference/python_vehicle_app_sdk/)
```

Comment:
The SDK link doesn't work, please correct by repaying the reference by concept.
- [The correct link](https://eclipse-velocitas.github.io/velocitas-docs/docs/concepts/python_vehicle_app_sdk/)